test/js/third_party/prisma/helper.ts

```diff
@@ -55,7 +55,7 @@ export function generate(type: string) {
 
   fs.writeFileSync(schema, content);
 
-  const result = Bun.spawnSync([bunExe(), "prisma", "generate", "--schema", schema], {
```

Comment:
using `bun x` shouldn't be necessary here

Reply:
Yeah, had meant to remove that, thanks!